Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Beckn/Select.hs

```diff
@@ -73,8 +73,8 @@ handler merchant sReq estimate = do
           driverSelectedFare = Nothing,
           customerExtraFee = sReq.customerExtraFee
         }
-  searchTry <- buildSearchTry merchantId searchReq.id estimate sReq.customerExtraFee sReq searchReq.estimatedDistance searchReq.estimatedDuration
   let estimateFare = fareSum fareParams
+  searchTry <- createNewSearchTry searchReq
```

Comment:
We need to update the searchRequest expiry time for both BAP and BPP after a retry.